### PR TITLE
Update developer guide to include user stories

### PR DESCRIPTION
**Pull request by TeddYE**

### CODE REVIEW

**Review by DALIN-Prog — CHANGES_REQUESTED** [review]
There is some merging conflict that you need to resolve. Have you tried to sync your master branch with the team repo's master branch?

**Comment by TeddYE** [comment]
Oops, hope I have managed to resolve it.

File: docs/AboutUs.md [hunk 1]
```diff
@@ -18,4 +18,14 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 * Role: Developer
 * Responsibilities: Data
 
+<<<<<<< HEAD
```
Comment (DALIN-Prog) [inline docs/AboutUs.md]: Think you have forgotten to remove this line

Comment (TeddYE) [reply docs/AboutUs.md]: Amended and update my part for user guide as well

**Review by DALIN-Prog — APPROVED** [review]

**Pull request merged**